### DISCUSSION TITLE
Is it possible to get any of the file source back from the exe file?

### DISCUSSION THREAD

**Discussion by 64ZooLane** [vercel/vercel#4902] [Help] [answered]
I have an old project which I used pkg to compile into an exe file for windows, but I can't seem to find the source anywhere. Im wondering what language it's in to try and decompile it or something. Any help would be great, cheers.

**Comment by styfle** [comment, marked as answer]
The `.exe` is a native binary on Windows.

Take a look at the available targets here: https://github.com/vercel/pkg#targets

**Reply by 64ZooLane** [reply]
I mean, how would I go about getting any readable code back from the exe file?

**Reply by 64ZooLane** [reply]
@styfle

**Reply by styfle** [reply]
Try this: https://stackoverflow.com/a/273248/266535
